language/nb.json

```
@@ -0,0 +1,146 @@
+{
+  "semantics": [
+    {
+      "label": "Oppgavebeskrivelse",
+      "description": "En beskrivelse av hvordan oppgaven skal løses."
+    },
+    {
+      "label": "Kildespråk",
+      "description": "Velg språk for kildeordene.",
+      "options": [
+        { "label": "Tysk" },
+        { "label": "Engelsk" },
+        { "label": "Fransk" },
+        { "label": "Norsk bokmål" },
+        { "label": "Norsk nynorsk" },
+        { "label": "Spansk" }
+      ]
+    },
+    {
+      "label": "Målspråk",
+      "description": "Velg språk for målordene.",
+      "options": [
+        { "label": "Tysk" },
+        { "label": "Engelsk" },
+        { "label": "Fransk" },
+        { "label": "Norsk bokmål" },
+        { "label": "Norsk nynorsk" },
+        { "label": "Spansk" }
+      ]
+    },
+    {
+      "label": "Ord",
+      "description": "Legg til ord ved å laste opp en CSV-fil eller skriv ordene inn i tekstfeltet.",
+      "important": {
+        "description": "<ul><li>Kilde- og målord er separert med komma (,).</li><li>Alternative svar er separert med en fremoverlent skråstrek (/).</li><li>Du kan legge til et teksttips ved å bruke kolon (:) foran tipset.</li></ul>",
+        "example": "water/sea:w___r,vann/hav:v__n"
+      }
+    },
+    {
+      "label": "Tilbakemelding på hele oppgava",
+      "fields": [
+        {
+          "widgets": [
+            {
+              "label": "Forhandsinnstilling"
+            }
+          ],
+          "label": "Opprett poengområder og legg inn tilbakemeldinger.",
+          "description": "Klikk på knappen \"Legg til poengområde\" og legg til så mange poengområder du trenger. Eksempel: 0-40 % Svakt resultat, 41-80 % Gjennomsnittlig resultat, 81-100 % Flott resultat!",
+          "entity": "Område",
+          "field": {
+            "label": "Tilbakemelding på hele oppgava",
+            "fields": [
+              {
+                "label": "Poengområde"
+              },
+              {
+                "label": ""
+              },
+              {
+                "label": "Tilbakemelding for definert poengområde",
+                "placeholder": "Skriv inn tilbakemelding."
+              }
+            ]
+          }
+        }
+      ]
+    },
+    {
+      "label": "Lokalisering",
+      "fields": [
+        {
+          "label": "Ingen gyldige ord",
+          "default": "Ingen gyldige ord funnet. Kontroller ordene dine og prøv igjen."
+        },
+        {
+          "label": "Åpne innstillinger aria tekst",
```

I don't think `aria` is anything users will know about.
```suggestion
          "label": "Åpne innstillinger (vises kun for skjermlesere)",
```